thesis-de/content-de.tex

```diff
@@ -110,6 +110,14 @@ \subsection{Datenerfassungssysteme}
 \subsection{Datenbanksysteme}
 
 \subsection{Frontends}
+Die Weboberfläche muss ausgewählten Benutzern eine Übersicht über die Nutzung einzelner Virtuellen oder Physische Maschinen bereitstellen.
+Hierbei ist wichtig, das die Sicherheit der Daten gewährleistet wird. Dies bedeutet, das sowohl die Verbindung zwischen Datenbank
```

Comment:
What kind of security do we want to ensure?
remember the three As:
* Authentication
* Authentication
* Authorisation